modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -257,6 +259,20 @@ public String getSearchInputName() {
 		return _searchInputName;
 	}
 
+	public String getSearchResultsTitle() {
+		if (_searchResultsTitle == null) {
+			HttpServletRequest httpServletRequest = getRequest();
+
+			ThemeDisplay themeDisplay =
+				(ThemeDisplay)httpServletRequest.getAttribute(
+					WebKeys.THEME_DISPLAY);
+
+			return LanguageUtil.get(themeDisplay.getLocale(), "search-results");
+		}
+
+		return _searchResultsTitle;
+	}
+
```

Comment:
This code correctly handles attribute and default, but:
1. We are not checking for `_managementToolbarDisplayContext`
2. We should be always internally translating label, to be consistent with other tags, like [for example here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java#L202-L211).
3. The default can be in `ManagementToolbarDefaults.java`

With above, the solution should be something like:
```java
public String getSearchResultsTitle() {
	String searchResultsTitle = _searchResultsTitle;

	if (searchResultsTitle == null) {
		if (_managementToolbarDisplayContext != null) {
			searchResultsTitle = _managementToolbarDisplayContext.getSearchResultsTitle();
		}

		if (searchResultsTitle == null) {
			searchResultsTitle = ManagementToolbarDefaults.getSearchResultsTitle();
		}
	}

	return LanguageUtil.get(
		TagResourceBundleUtil.getResourceBundle(pageContext),
		searchResultsTitle);
}

// in ManagementToolbarDefaults.java;
public static String getSearchResultsTitle() {
	return "search-results";
}
```

Reply:
Ahhhh that is much better, I was thinking if I should use LanguageUtil inside the defaults class, but it looked weird.

And yes, Fridays are very dangerous for reviewing code :laughing: